themes/custom/apigee_kickstart/src/components/form/fieldset.twig

```diff
@@ -25,11 +24,11 @@
   ]
   %}
   {#  Always wrap fieldset legends in a SPAN for CSS positioning. #}
-  <div class="card-header bg-white border-0 pt-4 pb-3">
+  <div class="card-header bg-white">
```

Comment:
This is still problematic. `<legend>` needs to be the direct child of `<fieldset>` to be valid.  If you throw this HTML in a validator, you'll get an error:

<img width="739" alt="Screen Shot 2019-07-09 at 11 10 18 AM" src="https://user-images.githubusercontent.com/60979/60900431-28556a00-a23a-11e9-8435-a255f49f24dc.png">


```
<fieldset data-drupal-selector="edit-payment-information-payment-method" id="edit-payment-information-payment-method--wrapper" class="fieldgroup form-composite js-form-item form-item js-form-wrapper form-wrapper card mb-3">
  <div class="card-header bg-white">
    <legend class="m-0">
      <span class="fieldset-legend mb-0">
      Payment method
      </span>
      <button class="btn-link">
        <svg class="svg-inline--fa fa-chevron-down fa-w-14 d-md-none" aria-hidden="true" focusable="false" data-prefix="fas" data-icon="chevron-down" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" data-fa-i2svg="">
          <path fill="currentColor" d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"></path>
        </svg>
      </button>
    </legend>
  </div>
  <div class="card-body p-4 fieldset-wrapper">
    <div id="edit-payment-information-payment-method" class="form-radios">
      <div class="js-form-item form-item js-form-type-radio form-item-payment-information-payment-method js-form-item-payment-information-payment-method form-check">
        <input class="payment-method--stored form-radio form-check-input" data-drupal-selector="edit-payment-information-payment-method-1" type="radio" id="edit-payment-information-payment-method-1" name="payment_information[payment_method]" value="1" checked="checked">
        <svg class="svg-inline--fa fa-dot-circle fa-w-16" aria-hidden="true" focusable="false" data-prefix="fas" data-icon="dot-circle" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" data-fa-i2svg="">
          <path fill="currentColor" d="M256 8C119.033 8 8 119.033 8 256s111.033 248 248 248 248-111.033 248-248S392.967 8 256 8zm80 248c0 44.112-35.888 80-80 80s-80-35.888-80-80 35.888-80 80-80 80 35.888 80 80z"></path>
        </svg>
        <svg class="svg-inline--fa fa-circle fa-w-16" aria-hidden="true" focusable="false" data-prefix="far" data-icon="circle" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" data-fa-i2svg="">
          <path fill="currentColor" d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm0 448c-110.5 0-200-89.5-200-200S145.5 56 256 56s200 89.5 200 200-89.5 200-200 200z"></path>
        </svg>
        <label class="form-check-label option" for="edit-payment-information-payment-method-1">American Express ending in 0002</label>
      </div>
      <div class="js-form-item form-item js-form-type-radio form-item-payment-information-payment-method js-form-item-payment-information-payment-method form-check">
        <input class="payment-method--new form-radio form-check-input" data-drupal-selector="edit-payment-information-payment-method-new-credit-card-authorize-net" type="radio" id="edit-payment-information-payment-method-new-credit-card-authorize-net" name="payment_information[payment_method]" value="new--credit_card--authorize_net">
        <svg class="svg-inline--fa fa-dot-circle fa-w-16" aria-hidden="true" focusable="false" data-prefix="fas" data-icon="dot-circle" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" data-fa-i2svg="">
          <path fill="currentColor" d="M256 8C119.033 8 8 119.033 8 256s111.033 248 248 248 248-111.033 248-248S392.967 8 256 8zm80 248c0 44.112-35.888 80-80 80s-80-35.888-80-80 35.888-80 80-80 80 35.888 80 80z"></path>
        </svg>
        <svg class="svg-inline--fa fa-circle fa-w-16" aria-hidden="true" focusable="false" data-prefix="far" data-icon="circle" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" data-fa-i2svg="">
          <path fill="currentColor" d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm0 448c-110.5 0-200-89.5-200-200S145.5 56 256 56s200 89.5 200 200-89.5 200-200 200z"></path>
        </svg>
        <label class="form-check-label option" for="edit-payment-information-payment-method-new-credit-card-authorize-net">Credit card</label>
      </div>
    </div>
  </div>
</fieldset>
```

Reply:
Thanks for running the validator @jacine. This made me looked into this a bit more. So the `.card` display is using `flexbox` and I found an opened issue where Chrome does not support `flexbox` for `<fieldset>`: https://bugs.chromium.org/p/chromium/issues/detail?id=375693

I replaced the card display with some custom classes and css. I thinks this looks better now. (This also cleans up some of the `.card-header` and `.card-body` overrides I had for commerce forms).

![Chris_Lastname___Apigee_Inc](https://user-images.githubusercontent.com/124599/60950290-eabf1280-a307-11e9-9304-e3acc8cc39fb.jpg)

Reply:
> So the .card display is using flexbox and I found an opened issue where Chrome does not support flexbox for <fieldset>: https://bugs.chromium.org/p/chromium/issues/detail?id=375693

Yes, thanks!  I starred the bug.  This is an old bug and somehow there are only 102 stars. ;)